src/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0 ;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+}
+
+.container__image {
+  display: inline-flex;
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 40px;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+.container__title {
+  padding: 0 16px 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  align-self: auto;
+  color: #060b35;
+}
+
+.container__text {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+  font-style: normal;
+
+  padding: 0 90px 16px 16px;
+}
+
+/* #region Stars */
+.container__review {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: space-between;
+  padding: 0 17px 24px 16px;
+  align-items: center;
+
+}
+
+.stars {
+  display: inline-flex;
+}
+
+.stars__star {
+  align-items: center;
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
Add the possibility to change active stars to 1, 2 and so on, by changing modifier. Not only option for 4 should exist(you can take this code from `stars` task
![Screenshot 2022-07-19 at 08 12 23](https://user-images.githubusercontent.com/17098736/179670010-8fa7cf66-362a-439d-9402-79701d532c11.png)
.